src/routes/utility.router.ts

```diff
@@ -0,0 +1,15 @@
+import Router from "koa-router";
```

Comment:
The term "utility" doesn't sound right. I'd instead suggest "miscellaneous" until we find the right buckets for each of these routes.

Reply:
Addressed